### PR TITLE
Remove branchff script

### PR DESCRIPTION
**Pull request by saschagrunert**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

We can remove the branchff script now since `krel ff` is in-place.

**Which issue(s) this PR fixes**:

Refers to https://github.com/kubernetes/sig-release/pull/998

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
- Removed `branchff` script in favor of `krel ff`
```

### CODE REVIEW

File: anago [hunk 1]
```diff
@@ -1138,7 +1138,7 @@ Additional binary downloads are linked in the [%s](%s).'
 ##############################################################################
 # When creating new alphas on the master branch, make sure that X.Y-1.0
 # has been created first or warn.
-# This is to ensure that branchff can continue to be used as needed.  User
+# This is to ensure that `krel ff` can continue to be used as needed.  User
 # can override here.
 # @param buildver - Incoming JENKINS_BUILD_VERSION
 branchff_sanity_check () {
```
Comment (justaugustus) [inline anago]: `s/branchff_sanity_check/fastforward_check`

File: docs/branching.md [hunk 2]
```diff
@@ -14,11 +14,11 @@ release branch to the HEAD of the master branch:
 *NOTE:* While the new release branch is in this midway state, no alpha
 releases should be created off the master branch as this would result in new
 master branch tags landing on the release branch.  The tooling in both anago
-and in branchff will catch this case, however.  This note is simply an FYI.
+and in `krel ff` will catch this case, however.  This note is simply an FYI.
```
[the remark this answers is not in the document]
Comment (saschagrunert) [reply docs/branching.md]: Sure, deleted the whole document :+1:

File: anago (hunk 1 above)
Comment (saschagrunert) [reply anago]: Renamed it :+1:

**Comment by saschagrunert** [comment]
Implemented your review suggestions :innocent:

**Pull request merged**